data/helpers.d/backend

```diff
@@ -154,7 +154,18 @@ ynh_remove_nginx_config () {
 #
 # usage: ynh_add_fpm_config
 ynh_add_fpm_config () {
-	finalphpconf="/etc/php5/fpm/pool.d/$app.conf"
+	local debian_release=$(lsb_release --codename --short)
```

Comment:
Can we made a small helper with that, in order to help the packagers to handle the migration in their app if needed ?

Reply:
Like just a `get_debian_release` that returns that single line...?

Reply:
Yes, really just a simple helper with this line.
Just to bring a little abstraction to our packagers